Tutorials/Multisite-Setup/index.md

```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+meta.Title: "Multisite setup in Umbraco"
+product: "CMS"
+meta.Description: "A guide to multisite setup in Umbraco"
+---
+
+# Multisite setup on Umbraco Cloud
+
+This tutorial explains how to host multiple sites from one project/installation of Umbraco.
+For practical reasons, we recommend using [Baselines](../../Umbraco-Cloud/Getting-Started/Baselines/) on the Cloud projects.
```

Comment:
Perhaps we could add a bit more context, as to why we recommend using the Baseline feature for Cloud projects? Could be something about the limitations to hostnames, content nodes or resources in general? 🤔

Reply:
noted